plugins/vst_base/CMakeLists.txt

```diff
@@ -27,33 +27,27 @@ SET(EXTERNALPROJECT_CMAKE_ARGS
 	"-DCMAKE_RUNTIME_OUTPUT_DIRECTORY_DEBUG=${CMAKE_RUNTIME_OUTPUT_DIRECTORY}"
 	"-DCMAKE_MODULE_PATH=${CMAKE_MODULE_PATH}"
 	"-DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}"
+	"-DCMAKE_TOOLCHAIN_FILE=${CMAKE_TOOLCHAIN_FILE}"
```

Comment:
This needs to be moved to the `MSVC`-specific part. It conflicts with the toolchain file passed in line 70.